rubygems.js

```diff
@@ -28,3 +28,19 @@ export async function rubygemsUpdate(rubygemsVersionInput, rubyPrefix) {
 
   return true
 }
+
+async function rubygemsLatest(gem, platform, engine, version) {
+  if (engine === 'ruby') {
+    if (version >= '3.0') {
```

Comment:
Comparing versions as strings feels brittle, could you use common.floatVersion() instead?

Reply:
Expecting a `x.10` version?  Regardless, I'll update.